garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/eyeposer.lua

```diff
@@ -28,7 +28,7 @@ local function ConvertRelativeToEyesAttachment( ent, pos )
 	local attachment = ent:GetAttachment( eyeattachment )
 	if ( !attachment ) then return end
 
-	local LocalPos, _ = WorldToLocal( pos, Angle( 0, 0, 0 ), attachment.Pos, attachment.Ang )
+	local LocalPos = WorldToLocal( pos, Angle( 0, 0, 0 ), attachment.Pos, attachment.Ang )

```

Comment:
ree don't recreate angle

Reply:
I'm not sure how I would do that.

Reply:
I don't know what he means either, but I'm assuming he means replace Angle(0,0,0) with angle_zero.

Reply:
yes